oneplusmvp/settings.py

```diff
@@ -182,10 +183,20 @@ def abspath(*args):
 )
 
 # Base url for vumi requests
-VUMI_GO_BASE_URL = "http://go.vumi.org/api/v1/go/http_api_nostream"
-VUMI_GO_CONVERSATION_KEY = "d5d981d5b86c4fee99f6ee078d0a0abd"
-VUMI_GO_ACCOUNT_KEY = "b365f245538841a08586a29b5b568c6c"
-VUMI_GO_ACCOUNT_TOKEN = "vaiToa6c"
+# VUMI_GO_BASE_URL = "http://go.vumi.org/api/v1/go/http_api_nostream"
+# VUMI_GO_CONVERSATION_KEY = "d5d981d5b86c4fee99f6ee078d0a0abd"
+# VUMI_GO_ACCOUNT_KEY = "b365f245538841a08586a29b5b568c6c"
+# VUMI_GO_ACCOUNT_TOKEN = "vaiToa6c"
```

Comment:
I don't know why these details were ever committed into a public repo. Instead of commenting out, please delete